service_info_cms/views.py

```diff
@@ -0,0 +1,31 @@
+from cms.models.pagemodel import Page
+from django.core.paginator import Paginator, EmptyPage, PageNotAnInteger
+from django.shortcuts import redirect, render
+from .models import PageRating
+
+
+def update_page_rating(request):
+    if request.method == 'POST':
+        rating = int(request.POST.get("rating", ""))
+        page_id = int(request.POST.get("page_id", ""))
+        return_url = request.POST.get("return_url", "")
```

Comment:
catch any bad input and do something simple like redirect to /  ;  can let that be the fallback if any input is bad or method != POST